cogboard-webapp/src/components/Widget/index.js

```diff
@@ -42,8 +42,10 @@ const Widget = ({ id, index }) => {
   const theme = useTheme();
   const [dialogOpened, openDialog, handleDialogClose] = useDialogToggle();
   const ref = useRef(null);
+  const isAuthenticated = useSelector(({app}) => !!app.jwToken);
```

Comment:
I would add spaces between destructuring curly braces in parameter function for consistency